### DISCUSSION TITLE
Customer Management Framework installation

### DISCUSSION THREAD

**Discussion by Sam-Valoxxxx** [pimcore/pimcore#15441] [Q&A / Support]
Hey,
i wanted to start using Pimcore for some of my projects. I saw that there is a prebuilt Customer Management Framework so i tried to install it on Pimcore 11.0.2. but i'm encountering an Error. Here is the command i was using:

`composer update pimcore/customer-management-framework-bundle --ignore-platform-req=ext-zip`

The error message I'm receiving is:

`Symfony\Component\ErrorHandler\Error\ClassNotFoundError: Attempted to load class "PimcoreCustomerManagementFrameworkBundle" from namespace "Pimcore\Bundle\CustomerManagementFrameworkBundle". Did you forget a "use" statement for another namespace?
`

I'm running this command as a root/super user. Can someone please help me understand and resolve this issue?

**Comment by johanlindgren** [comment]
Require the bundle as www-data  using
```
composer require pimcore/customer-management-framework-bundle
```
and then make sure to enable it in config/bundles.php with 
```
<?php
return [
     CustomerManagementFrameworkBundle\PimcoreCustomerManagementFrameworkBundle::class => ['all' => true],
     Pimcore\Bundle\ObjectMergerBundle\ObjectMergerBundle::class => ['all' => true],
];
```
before you install it using
```
 bin/console pimcore:bundle:install PimcoreCustomerManagementFrameworkBundle
```
